tests/test_fxdc_core.py

```
@@ -1,49 +1,55 @@
 import pytest
-from fxdc import load, loads, dumps, to_json, Config
-from fxdc.exceptions import FXDCException, InvalidData
+
+from fxdc import dumps, loads, to_json
+from fxdc.exceptions import InvalidData
+
 
 def test_basic_primitives_roundtrip():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_basic_primitives_roundtrip`

Add return type annotation: `None`

(ANN201)